pkgs/development/python-modules/aiocoap/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, buildPythonPackage
+, cbor2
+, cryptography
+, fetchFromGitHub
+, filelock
+, hkdf
+, linkheader
+, pynacl
+, python
+, sqlalchemy
+, uvloop
+, websockets
+}:
+
+buildPythonPackage rec {
+  pname = "aiocoap";
+  version = "0.4b3";
+
+  src = fetchFromGitHub {
+    owner = "chrysn";
+    repo = pname;
+    rev = version;
+    sha256 = "1zjg475xgvi19rqg7jmfgy5nfabq50aph0231p9jba211ps7cmxw";
+  };
+
+  # Extra dependency DTLSSocket is not present
+  propagatedBuildInputs = [
+    cbor2
+    cryptography
+    filelock
+    linkheader
+    pynacl
+    websockets
+  ];
+
+  checkInputs = [
+    cbor2
+    hkdf
+    linkheader
+    uvloop
+    websockets
```

Comment:
dependencies that are listed as propagatedBuildInputs shouldn't need to be repeated as checkInputs

Reply:
Thanks for the feedback. I will check that.

Reply:
`checkInputs` are no longer needed.